ans2ask-ui/src/components/Login/Login.css

```diff
@@ -4,8 +4,8 @@
 }
 
 .custom-margin {
-    margin-top: 185px;
-    margin-bottom: 185px;
+    margin-top: 173px;
```

Comment:
Let's also add more spacing in between the Login button and forgot passwords/don't have an account links. Let's try out 185-190, since it looks a little bunched together.

Reply:
the custom-margin refers to the margin of the full div/form, but I know what you mean, I adjusted the margin between the login button and the links:

 <div className="form-group mb-4">        ----->>      <div className="form-group mb-5">